spec/helper/MockHelper.ts

```diff
@@ -359,8 +358,8 @@ class PlayerEventHelper implements EventEmitter {
       type: PlayerEvent.AdSkipped,
       ad: {
         isLinear: true,
-        width: null,
-        height: null,
+        width: 0,
```

Comment:
Just make it types-compatible (it cannot be `null` according to the latest types).